modules/beetmover_scriptworker/files/requirements.txt

```diff
@@ -6,7 +6,7 @@ aiohttp==3.4.4
 arrow==0.12.1
 async_timeout==3.0.0
 attrs==18.2.0
-beetmoverscript==7.9.1  # puppet: nodownload
+beetmoverscript==8.0.0.dev0  # puppet: nodownload
```

Comment:
For the record, this is temporary and will go away before merging. I just use this as it's easier to edit on my local machine and pull in my puppet environment.